src/components/Message/hooks/useReactionHandler.js

```diff
@@ -48,28 +48,29 @@ export const useReactionHandler = (message) => {
     - Make the API call in the background
     - If it fails, revert to the old message...
      */
-    if (userExistingReaction) {
-      reactionChangePromise = channel.deleteReaction(
-        message.id,
-        // @ts-ignore Typescript doesn't understand that the userExistingReaction variable might have been mutated inside the foreach loop
-        userExistingReaction.type,
-      );
-    } else {
-      // add the reaction
-      const messageID = message.id;
+    if (message.id) {
```

Comment:
same goes for this extra check

Reply:
Yeah what is optional and not could use another sweep through in the stream chat js, this one is cause when a message is not a response id is optional and they get intermingled in use. Dan and I chose to be conservative where we weren't 100% and list things as optional to reduce the risk of exceptions.

Reply:
Staying conservative it seems...